src/content/changes/highlights/2021-11-19.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2021-11-19T10:00:00
+title: Highlights for the week ending November 19, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+AWS LB Controller is now available! Find it in the managed apps catalog.
+
+oauth2-proxy version v7.2.0 is now available, as well as support for configuring extra arguments (per provider, or for all providers).
+
+## kubectl gs
+
+The flag --self-contained is now available when logging in, and will output a standalone kubeconfig file.
+
+You can now specify a Service Account token when logging in, to help automation use cases.
```

Comment:
```suggestion
You can now specify a Service Account token when logging in, to help automation use cases.

## Documentation

Added [documentation](https://docs.giantswarm.io/advanced/cost-optimization/workload-clusters-scale-down/) on how to scale down workload clusters and silence alerts for a specified period of time. This can be used, for example, to limit workload clusters costs.
```